api/desecapi/models.py

```diff
@@ -472,7 +472,7 @@ class Meta:
 # restricted types are managed in use by the API, and cannot directly be modified by the API client
 RR_SET_TYPES_AUTOMATIC = {
     # corresponding functionality is automatically managed:
-    'CDNSKEY', 'CDS', 'DNSKEY', 'KEY', 'NSEC', 'NSEC3', 'OPT', 'RRSIG',
+    'KEY', 'NSEC', 'NSEC3', 'OPT', 'RRSIG',
```

Comment:
we should make sure somewhere that deleting DNSKEY records leaves the automatically provisioned ones intact

Reply:
done